components/PictureList.js

```diff
@@ -0,0 +1,42 @@
+import { Fragment } from "react";
+import Image from "next/image";
+import { styled } from "styled-components";
+import pictures from "../lib/db.js";
+
+export default function Picture() {
+  return (
+    <>
+      <StyledArticle>
+        {pictures.map((picture) => (
+          <>
```

Comment:
@DorisW3  recommend to not place an empty fragment right after mapping. Do it like so: 
`<Fragment key={picture.id}>
            <StyledImage
              src={picture.image}
              alt={picture.theme}
            /></Fragment>`

Try to use `<ul key={picture.id}><li><StyledImage src={} alt={} height={} width={}/></li></ul>` - Semantik it is better accessable.